alsa.te

```diff
@@ -95,29 +104,6 @@ userdom_manage_unpriv_user_semaphores(alsa_t)
 userdom_manage_unpriv_user_shared_mem(alsa_t)
 userdom_search_user_home_dirs(alsa_t)
 
-ifdef(`distro_debian',`
-	term_dontaudit_use_unallocated_ttys(alsa_t)
-
-	# Gnome 3.4 bug
-	dev_associate(alsa_tmpfs_t)
-
-	allow alsa_t self:capability kill;
-
-	manage_lnk_files_pattern(alsa_t, alsa_var_lib_t, alsa_var_lib_t)
-	files_root_filetrans(alsa_t, alsa_var_lib_t, dir, ".config")
-
-	fs_list_tmpfs(alsa_t)
-
-	optional_policy(`
-		dbus_read_lib_files(alsa_t)
-	')
-
-	optional_policy(`
-		pulseaudio_run(alsa_t, system_r)
-		pulseaudio_tmpfs_content(alsa_tmpfs_t)
-	')
-')
```

Comment:
@pebenito I checked again, the initscript is gone (only kept as an example file somewhere in the pkg) and not in /etc/init.d anymore. so maybe this could be removed or a boolean should maybe added